### PR TITLE
[Jolokia Input] Add processors option // do not approve

### LINKED ISSUE #10033: [Jolokia Input] Add processors option
Add the processors option to Jolokia Input integration.

image

### PR DESCRIPTION
**Pull request by MatheusGelinskiPires**
## Proposed commit message

Add the processors option to Jolokia Input integration.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- Used the kubernetes integration as reference to add the processors option: [data stream apiserver](https://github.com/elastic/integrations/tree/main/packages/kubernetes/data_stream/apiserver).

## How to test this PR locally

1. Install Elastic Agent.
2. Add Jolokia Input integration.
3. Set up a processor.

### CODE REVIEW

**Comment by MatheusGelinskiPires** [comment]
contributor agreement done

**Comment by MatheusGelinskiPires** [comment]
> ❌ Author of the following commits did not sign a [Contributor Agreement](https://www.elastic.co/contributor-agreement): [a987736](https://github.com/elastic/integrations/commit/a98773621185c8aec31792401d00073103f63951), [29caf4b](https://github.com/elastic/integrations/commit/29caf4b9e4df4e399a9024f80fbb905abe32c37b), [90e3436](https://github.com/elastic/integrations/commit/90e34367b3fe5bdaa52f5f40ab9a26e07ea2a00c), [db908ff](https://github.com/elastic/integrations/commit/db908ff51b860572a1321cc1d7400e88c7212a27)
> 
> Please, read and sign the above mentioned agreement if you want to contribute to this project

done

**Comment by MatheusGelinskiPires** [comment]
Please label the PR as an enhancement as I have no permissions to set labels.

File: packages/jolokia_input/manifest.yml [hunk 1]
```diff
@@ -60,6 +60,15 @@ policy_templates:
...
               - attr: NonHeapMemoryUsage
                 field: memory.non_heap_usage
+      - name: processors
+        type: yaml
+        title: Processors
+        multi: false
+        required: false
+        show_user: false
+        description: >
+          Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the events are shipped. See [Processors](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html) for details.
```
Comment (aliabbas-elastic) [inline packages/jolokia_input/manifest.yml:70]:
```suggestion
          Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the events are shipped. See [Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) for details.
```

File: packages/jolokia_input/manifest.yml [hunk 2]
```diff
@@ -60,6 +60,15 @@ policy_templates:
...
+
```
Comment (aliabbas-elastic) [inline packages/jolokia_input/manifest.yml:71]: Nit:- empty line

File: packages/jolokia_input/changelog.yml [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # newer versions go on top
-- version: 0.3.0
+- version: "0.3.1"
```
Comment (aliabbas-elastic) [inline packages/jolokia_input/changelog.yml:2]: Need to update this version in main `manifest.yml`

**Review by aliabbas-elastic — CHANGES_REQUESTED** [review]

**Comment by aliabbas-elastic** [comment]
@MatheusGelinskiPires We need to run the CI once on your changes and that can be done once the CLA check is passed

**Comment by MatheusGelinskiPires** [comment]
> @MatheusGelinskiPires We need to run the CI once on your changes and that can be done once the CLA check is passed

working on it

**Comment by MatheusGelinskiPires** [comment]
hello @aliabbas-elastic , seems that I have some misconfiguration at my side (git cli).
I will try to add another pull request from a different branch to see if the CLA will works

**Comment by MatheusGelinskiPires** [comment]
Closing this PR as it was replaced by #10068

**Pull request closed without merging**